bin/generate_contributors.go

```diff
@@ -120,6 +90,37 @@ func main() {
 			}
 			page++
 		}
+
+		// Get all created PR contributors
+		prRangeOk = true
+		page = 0
+
+		for page < 15 && prRangeOk{
+			repoOpts := &github.PullRequestListOptions{State: "all", ListOptions: github.ListOptions{PerPage: 100, Page: page}}
+			prs, _, err := client.PullRequests.List(ctx, "sourcegraph", repo, repoOpts)
+			if err != nil {
+				log.Fatal("error fetching pull requests", err)
+				return
+			}
+
+			if len(prs) == 0 {
+				break
+			}
+
+			for _, pr := range prs {
+				
+				if pr.CreatedAt.Before(issueDate) {
+					prRangeOk = false
```

Comment:
You can label loops and break out of multiple, e.g.

```go
outer: for page < 15 {
   for {
       break outer
    }
}
```

Reply:
🤯  so cool!